packages/video_player/video_player_avfoundation/pigeons/messages.dart

```diff
@@ -81,4 +85,6 @@ abstract class AVFoundationVideoPlayerApi {
   void pause(TextureMessage msg);
   @ObjCSelector('setMixWithOthers:')
   void setMixWithOthers(MixWithOthersMessage msg);
+  @ObjCSelector('setupAudioSession:')
+  void setupAudioSession(AudioSessionMessage msg);
```

Comment:
Nit: "setup" is a noun; methods should use verbs, and the verb is "set up", so this should be "setUp". Applies throughout the PR.